src/System.Collections/tests/Generic/Dictionary/Dictionary.Tests.netcoreapp.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Xunit;
+
+namespace System.Collections.Tests
+{
+    public partial class Dictionary_IDictionary_NonGeneric_Tests : IDictionary_NonGeneric_Tests
+    {
+        protected override ModifyOperation ModifyEnumeratorAllowed => ModifyOperation.Remove;
```

Comment:
Nit, the ordering of ModifyEnumeratorAllowed and ModifyEnumeratorThrows  is different in the other files.